new/community-slack.md

```diff
@@ -0,0 +1,170 @@
+world of clients (web or otherwise), social customs (pastebins etc), and overall
+usage that it entails. This has led to many Alexes unable to find the support
+they need to be successful with Chef. The flip is that the Bonnies of the world
+often don't want to deal with the usability nightmare that is IRC, and thus are
+not able to jump in and answer questions even when they would otherwise have the
+time and energy to do so. This has left the Bonnie role on the shoulders of
+relatively few individuals in an overall unsustainable way. Slack helps both
+Alex and Bonnie to reach their goals with a more modern experience and tools
+they are already familiar with.
+
+## Prior Art
+
+Some other communities have gone through similar transitions recently, with
+mixed results. The most high-profile negative experience was from [Free Code Camp](http://blog.freecodecamp.com/2015/06/so-yeah-we-tried-slack-and-we-deeply-regretted-it.html),
+who quickly hit problems around membership caps. [Reactiflux](https://facebook.github.io/react/blog/2015/10/19/reactiflux-is-moving-to-discord.html)
+had the same problem at around 7,500 users. The `#chef` IRC channel has had
+3,137 distinct usernames in the past year, so we aren't likely to immediately
+hit that kind of cap, but it persistent risk factor going forward.
+
+More positively, [Wordpress](https://make.wordpress.org/chat/) migrated from
+IRC to Slack and has been successful with over 8000 users. The HangOps community
+Slack has also been far more successful than it's prior IRC channel. The Elixir
+and Clojure communities have both started a similar migration, with around 5000
+users each in their Slack teams.
+
+Overall it seems like communities that have done similar IRC to Slack migrations
+have been happy with them, but those hitting hard limits on the Slack side have
+had zero recourse but to abort and move elsewhere.
+
+## Alternatives
+
+For the two problem cases mentioned above, they chose to migrate to [Gitter](https://gitter.im/)
+and [Discord](https://discordapp.com/) respectively. Gitter doesn't have the
+same level of mobile device support and might confuse some users as our
+support discussions generally happen in a single, shared channel rather than
+the smattering of repositories that make up Chef, Chef Server, etc. Discord
+seems reasonable as an alternative, but the market penetration of Slack is a
+strong nod in their favor.
+
+There are a few generally-similar FOSS options
+([Mattermost](http://www.mattermost.org/), [Zulip](https://www.zulip.org/), etc)
+but none seem to be well polished enough to be worth migrating to at this point.
+Additionally most require self-hosting, which would increase the operational
+load on Chef Software's staff.
+
+Another option is to simply stick with IRC, and try to set up a mix of hosted
+bouncers/proxies and an improved web client. While this is possible, I've not
+heard of any other major community providing this level of service for IRC.
+
```

Comment:
Could a StackOverflow chat room be another alternative ? 
The UX is not that bad, there's link preview and code formatting, no mail disclosure is needed and there's a kind of moderation available for room owners. (just my 2cts)